saleor/checkout/utils.py

```diff
@@ -880,8 +880,8 @@ def create_order(cart: Cart, tracking_code: str, discounts, taxes):
     return order
 
 
-def is_fully_paid(cart: Cart, taxes, discounts):
-    """Check if checkout is fully paid."""
+def can_be_fully_paid(cart: Cart, taxes, discounts):
+    """Check if checkout can be fully paid with proper payments."""
```

Comment:
I'd only change this comment to be more accurate. Maybe something like "Check if provided payment methods cover the checkout's total amount."

Reply:
OK let's only change the comments then.